packages/netlify-cms-backend-github/src/API.js

```diff
@@ -24,9 +24,9 @@ export default class API {
     this.useOpenAuthoring = config.useOpenAuthoring;
     this.repo = config.repo || '';
     this.originRepo = config.originRepo || this.repo;
-    this.repoURL = `/repos/${this.repo}`;
+    this.repoURL = this.repo ? `/repos/${this.repo}` : '';
```

Comment:
I think `repoURL` and `originRepoURL` will always be set to an empty string with `git-gateway` since `config.repo` is `undefined` as passed here: https://github.com/netlify/netlify-cms/blob/c1c95874f41bd2b435c39d3dea263c77eaa601eb/packages/netlify-cms-backend-git-gateway/src/implementation.js#L160.

Also the `repoURL` value is overwritten in the parent class when using `git-gateway` https://github.com/netlify/netlify-cms/blob/c1c95874f41bd2b435c39d3dea263c77eaa601eb/packages/netlify-cms-backend-git-gateway/src/GitHubAPI.js#L10